src/analysis/retail/monk/mistweaver/modules/spells/RisingSunKick.tsx

```diff
@@ -41,9 +41,6 @@ class RisingSunKick extends Analyzer {
 
     if (this.lastBOK > this.lastRSK) {
       this.rskResets += 1;
-
-      this.spellUsable.endCooldown(TALENTS_MONK.RISING_SUN_KICK_TALENT.id);
-      this.spellUsable.beginCooldown(event, TALENTS_MONK.RISING_SUN_KICK_TALENT.id);
```

Comment:
I thought you added these lines recently - is this a merge mistake?

Reply:
The first PR was on the shadowlands branch, but this is for DF branch. Is there some syncing across them that I should've been using?

Reply:
No - I might be getting mixed up I thought these lines were supposed to be added not removed

Reply:
Nope, they are what was causing the issue. It was restarting the CD here, but also getting restarted in SpellUsable.tsx which is what causes the issue.

Reply:
Apologies then, in you go